### ISSUE TITLE
Contrast themes – Tooltip-style dropdowns are unidentifiable in forced colors mode

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#8830]
In WHCM / forced colors mode, there are two issues with our tooltip-style "More" dropdowns:

- They are missing a border or outline, so it’s hard to tell where the dropdown is
- The tooltip arrow is a square

Screenshot:

![Image](https://user-images.githubusercontent.com/877585/178238918-7fa746b9-6a2a-4212-afe7-fa4e1a2f5b2f.png)

Figma link: https://www.figma.com/file/O81E48wieOZ7e6AivvksWn/Contrast-themes%3A-Wagtail-audit?node-id=6766%3A4147

---

## Border / outline

We should add a relatively thick border / outline so the dropdown is easy to identify. We could try a few different styles to figure out what feels right.

## Tooltip arrow

Discussing this on the [web-a11y Slack](https://web-a11y.slack.com/archives/C02D66WBDTQ/p1657203369545429), a few options were proposed:

- Use a unicode arrow (`content: '\25B2';`)
- Use a clip path (`clip-path: polygon(50% 0, 0 100%, 100% 100%).`)
- Use a background-image (but I would have expected this to not show in WHCM?)
- Use `forced-color-adjust: none;` (but we then lose the correct color from WHCM)

The best option seems to me like clip-path, or otherwise the unicode arrow. It would be worth trying a few just to be sure.

**Comment by lb-** [comment]
So that we have this in 4.0 I added the finishing touches to #8897 - here is my discovery on the arrow.

* unicode arrow was a pain to position correctly and could cause issues depending on font rendering
* background colour worked ok but felt hacky
* clip path worked well but still needed a specific background `ButtonText` on forced color

Went with the last one (clip path) so that the PR could be merged but we may want to revisit this solution in the future.

It would be good to note that this PR and the solution does not cover all dropdowns - as we have multiple implementations. I know there is planned work to unify these so for now will close this issue as fixed when that is merged.

Maybe @Scotchester  and @anujaraj  you can do a review of dropdowns again and we may need a new issue raised OR the other issues already cover these additional scenarios.

* https://github.com/wagtail/wagtail/issues/8835
* https://github.com/wagtail/wagtail/issues/8827
